### DISCUSSION TITLE
.npmrc file deleted after "npx projen"

### DISCUSSION THREAD

**Discussion by fl0wo** [projen/projen#2884] [Q&A] [answered]
Hi, is there a way to specify the `.npmrc` file content (or how it should be generated) directly in the` .projenrc.ts` file?

Problem: 
My file `.npmrc` gets deleted after running `npx projen`

Desired:
After running `npx projen`, still have the file in the root project

Using projen for `AwsCdkTypeScriptApp`

**Comment by mrgrain** [comment, marked as answer]
`project.npmrc.addConfig()` should do the trick

**Comment by fl0wo** [comment]
Sooo simple!

It worked thank you
